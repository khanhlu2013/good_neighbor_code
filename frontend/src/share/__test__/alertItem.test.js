import React from "react";
import renderer from "react-test-renderer";

import AlertItem from "../alertItem";

test("AlertItem can display count and isImportant of true", () => {
  const component = renderer.create(<AlertItem count={3} isImportant={true} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AlertItem can display count and isImportant of false", () => {
  const component = renderer.create(
    <AlertItem count={4} isImportant={false} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
