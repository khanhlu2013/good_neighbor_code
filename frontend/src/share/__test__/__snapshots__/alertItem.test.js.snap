// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertItem can display count and isImportant of false 1`] = `
<div
  className="sc-bdVaJa cLamRz"
>
  <div
    className="sc-bwzfXH bENTBn"
  >
    4
  </div>
</div>
`;

exports[`AlertItem can display count and isImportant of true 1`] = `
<div
  className="sc-bdVaJa cLamRz"
>
  <div
    className="sc-bwzfXH gMdCgE"
  >
    3
  </div>
</div>
`;
